db/install.xml

```diff
@@ -83,5 +83,29 @@
         <KEY NAME="groupid" TYPE="foreign" FIELDS="groupid" REFTABLE="groups" REFFIELDS="id"/>
       </KEYS>
     </TABLE>
+    <TABLE NAME="ratingallocate_choice_group" COMMENT="Maps Groups and the Allocations on which they are created with">
```

Comment:
I struggle with how similar the two tables are called (`ratingallocate_choice_group` vs `ratingallocate_group_choices`). As the other is already used by some users, maybe change this to `ratingallocate_allocationchoice_group`?

Reply:
It's not possible to change the name to something much different since tablenames can only be 28 characters long